<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mircotubule Project</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>580</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="conditionlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Condition: IDLE</string>
    </property>
   </widget>
   <widget class="QLabel" name="constant_sourceviewlabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>45</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="constant_processedviewlabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>45</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Processed Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="graphicslabel_Source">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="graphicslabel_Processed">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="graphicslabel_Original">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>70</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="constant_originalviewlabel">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>45</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Video</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionSelect_ROI"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionAbsort"/>
   </widget>
   <widget class="QMenu" name="menuPreview">
    <property name="title">
     <string>Preview</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuPreview"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSelect_ROI">
   <property name="text">
    <string>Select ROI</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionAbsort">
   <property name="text">
    <string>Absort</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
